infra/base-images/base-runner/bad_build_check

```diff
@@ -136,10 +133,7 @@ function check_startup_crash {
     SKIP_SEED_CORPUS=1 run_fuzzer $FUZZER_NAME -seed=1337 -runs=$MIN_NUMBER_OF_RUNS &>$FUZZER_OUTPUT
     CHECK_PASSED=$(egrep "Done $MIN_NUMBER_OF_RUNS runs" -c $FUZZER_OUTPUT)
   elif [[ "$FUZZING_ENGINE" = afl ]]; then
-    # TODO(https://github.com/google/oss-fuzz/issues/2470): Dont use
-    # AFL_DRIVER_DONT_DEFER by default, support .options files in
-    # bad_build_check instead.
-    AFL_DRIVER_DONT_DEFER=1 AFL_NO_UI=1 SKIP_SEED_CORPUS=1 timeout --preserve-status -s INT 20s run_fuzzer $FUZZER_NAME &>$FUZZER_OUTPUT
+    AFL_FORKSRV_INIT_TMOUT=30000 AFL_NO_UI=1 SKIP_SEED_CORPUS=1 timeout --preserve-status -s INT 35s run_fuzzer $FUZZER_NAME &>$FUZZER_OUTPUT
```

Comment:
Are we doing this on ClusterFuzz?
Or should we change things?

Reply:
AFL_DRIVER_DONT_DEFER is in src/python/tests/core/bot/fuzzers/afl/afl_launcher_test.py --- you know my problems running & debugging clusterfuzz so I didnt touch that. (as long as nobody complains it is not working ...)
AFL_FORKSRV_INIT_TMOUT is something that should be added to clusterfuzz I think. I can do a PR tomorrow

Reply:
ah the AFL_DRIVER_DONT_DEFER is just in a test, so doesnt apply.
PR for forkserver init timeout is in.